header-footer-grid/Core/Components/MenuIcon.php

```diff
@@ -150,9 +150,35 @@ public function add_menu_icon_classes( $class = '', $icon = '' ) {
 	public function load_scripts() {
 		if ( $this->is_component_active() || is_customize_preview() ) {
 			wp_add_inline_style( 'neve-style', $this->toggle_style() );
+			wp_add_inline_script( 'neve-script', $this->toggle_script() );
 		}
 	}
 
+	/**
+	 * Get JS to use as inline script
+	 *
+	 * @return string
+	 */
+	public function toggle_script() {
+		// Add aria-expanded to toggle function to be used when the menu icon is used,
+		$script = <<<JS
+function toggleAriaClick() {
```

Comment:
If we are focusing on small gains in script size, what do you think of adding the minified version of the script instead? The full version can be in method comments maybe? Minified will strip down some it to 60% of its current size.

Reply:
Good point, I've reused the minify for the CSS to remove the whitespace and reduce the overall size.